CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Release Notes for Craft Commerce
 
+
+## Unreleased
+
+- Added refundTransaction object to `\craft\commerce\services\Payments::EVENT_AFTER_CAPTURE_TRANSACTION` event. ([#2081](https://github.com/craftcms/commerce/issues/2081))
```

Comment:
is this the `EVENT_AFTER_CAPTURE_TRANSACTION` or `EVENT_AFTER_REFUND_TRANSACTION`?